test/device/test_accelerometer.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+from app.device.accelerometer import Accelerometer
+
+
+def test_not_moving():
+#sample data while not moving
+    raw_data = [18, 0, 4, 0, 234, 0]
+
+#parsed sample data
+    expected_data ={
+        'x_acceleration': 0.0702,
+        'z_acceleration': 0.9828,
+        'y_acceleration': 0.0156,
+        'time': 1518136213.101481

```

Comment:
pretty sure having the time hard coded will cause the test to fail in the future
we can use pytest to evluate a subset of keys in the dictionary [example](https://docs.pytest.org/en/latest/builtin.html#pytest.approx)